giskard/project.py

```diff
@@ -1,6 +1,8 @@
 import json
 import logging
 from typing import Callable, Dict, Iterable, List, Optional, Union
+import sys
+sys.tracebacklimit = None # set to 0 for No stack trace
```

Comment:
what was the reason for adding this?

Reply:
JM had suggested if the stacktrace will be confusing for users but we discussed and decided to keep the stack trace. So i am deleting this part .